### PR TITLE
Added .curlrc file

### PR DESCRIPTION
**Pull request by ghost**
Added some basic self explanatory options, to make curl more clear.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -7,6 +7,7 @@ echo "Creating backup files"
 [ -f ~/.vimrc ] && cp ~/.vimrc ~/.vimrc.orig.$now
 [ -f ~/.tmux.conf ] && cp ~/.tmux.conf ~/.tmux.conf.orig.$now
 [ -f ~/.zcompdump ] && cp ~/.zcompdump ~/.zcompdump.orig.$now
+[ -f ~/.zshrc ] && cp ~/.curlrc ~/.curlrc.orig.$now
```
Comment (ganglio) [inline install.sh]: here it checks for .zshrc and not .curlrc

Comment (ghost) [reply install.sh]: oops will change

File: vimrc [hunk 2]
```diff
@@ -9,6 +9,7 @@ Plugin 'sheerun/vim-polyglot'
 Plugin 'mattn/emmet-vim'
 Plugin 'scrooloose/nerdtree'
 Plugin 'christoomey/vim-tmux-navigator'
+Plugin 'scrooloose/nerdtree'
```
Comment (ganglio) [inline vimrc]: It was already there. Check two lines above that one :p

Comment (ghost) [reply vimrc]: been fixed.

**Comment by ghost** [comment]
Made changes.

**Pull request closed without merging**